Loop/Views/MicrobolusView.swift

```diff
@@ -11,31 +11,37 @@ import Combine
 import LoopCore
 
 struct MicrobolusView: View {
-    typealias Result = (microbolusesWithCOB: Bool, withCOBValue: Double, microbolusesWithoutCOB: Bool, withoutCOBValue: Double, safeMode: Microbolus.SafeMode)
+    typealias Result = (microbolusesWithCOB: Bool, withCOBValue: Double, microbolusesWithoutCOB: Bool, withoutCOBValue: Double, safeMode: Microbolus.SafeMode, microbolusesMinimumBolusSize: Double)
 
     final class ViewModel: ObservableObject {
         @Published var microbolusesWithCOB: Bool
         @Published var withCOBValue: Double
         @Published var microbolusesWithoutCOB: Bool
         @Published var withoutCOBValue: Double
         @Published var safeMode: Microbolus.SafeMode
+        @Published var microbolusesMinimumBolusSize: Double
 
         @Published fileprivate var pickerWithCOBIndex: Int
         @Published fileprivate var pickerWithoutCOBIndex: Int
+        @Published fileprivate var pickerMinimumBolusSizeIndex: Int
 
         fileprivate let values = stride(from: 30, to: 301, by: 5).map { $0 }
+        // @ToDo: Should be able to get the to limit from the settings but for now defult to a low value
+        fileprivate let minimumBolusSizeValues = stride(from: 0.0, to: 0.51, by: 0.05).map { $0 }
```

Comment:
Medtronic VEO has a bolus step 0.025 U

Reply:
I think you can leave it like that for now, and take it from the settings in the future.